src/interfaces/assistants_web/src/components/MessagingContainer/AssistantTools.tsx

```diff
@@ -41,18 +41,16 @@ export const AssistantTools: React.FC<{
             {availableTools.map(({ name, display_name, description, error_message }) => {
               const enabledTool = enabledTools.find((enabledTool) => enabledTool.name === name);
               const checked = !!enabledTool;
-              const disabled = !checkIsBaseAgent(agent);
 
               return (
                 <ToggleCard
                   key={name}
-                  disabled={disabled}
                   errorMessage={error_message}
                   checked={checked}
                   label={display_name ?? name ?? ''}
                   icon={getToolIcon(name)}
                   description={description ?? ''}
-                  onToggle={(checked) => handleToggle(name ?? '', checked)}
+                  onToggle={(checked) => handleToggle(name!, checked)}
```

Comment:
Obviously you don't have to be the one to fix this, but it's a bit surprising and challenging that `name` can be null in `ToolDefinition`

Reply:
Yeah there are a lot of fallbacks for various attributes in the client-side. Something we could refine by adding better type checking